api/identitatem/v1alpha1/authrealm_types.go

```diff
@@ -59,7 +62,17 @@ type LDAPExtraConfig struct {
 	// Optional filter to apply when searching the directory. For example "(objectClass=person)"
 	// +optional
 	Filter string `json:"filter,omitempty"`
+
+	//Optional A yaml representing the Dex GroupSearch parameters. (see https://github.com/dexidp/dex/blob/master/connector/ldap/ldap.go#L43)
```

Comment:
I think we should remove `dex` word and URL as we don't want to expose it to customer